src/tests/WebApi.OpenAPI.SystemTest/openapi-v1.yaml

```diff
@@ -42,4 +42,210 @@ paths:
       responses:
         '200':
           description: Success
-components: { }
\ No newline at end of file
+  '/minimal-endpoint-with-typed-result-no-produces/{id}':
```

Comment:
We should almost used our pipeline to detect breaking changes to OpenAPI.spec.

Reply:
good point, though, oasdiff pipeline is hosted on ADO and we don't have the equivalent on github reusable workflows